core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -3224,11 +3266,120 @@ secret_configs:
     - non_app_ci
     name: splunk-auth
     namespace: rh-corp-logging
+- from:
+    .dockerconfigjson:
+      dockerconfigJSON:
+      - auth_field: token_image-puller_build03_reg_auth_value.txt
+        item: build_farm
+        registry_url: image-registry.openshift-image-registry.svc.cluster.local:5000
+      - auth_field: token_image-puller_build03_reg_auth_value.txt
+        item: build_farm
+        registry_url: image-registry.openshift-image-registry.svc:5000
+      - auth_field: token_image-puller_build03_reg_auth_value.txt
+        item: build_farm
+        registry_url: registry.build03.ci.openshift.org
```

Comment:
We have not customized the domain of registry on build03, which is fine as long as we do not run jobs there.

Reply:
/hold
Let me to set it up.